### PR TITLE
CORE-17605 - Removed deprecated code related to `useAndRelease`

### PR DESCRIPTION
**Pull request by filipesoliveira**
The method `useAndRelease` has been deprecated and it will be removed in the near future. All references to the method were removed since the method is no longer relevant.

This PR depends on: https://github.com/corda/corda-api/pull/1289.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -46,7 +46,7 @@ commonsTextVersion = 1.10.0
 bouncycastleVersion=1.76
 # Corda API libs revision (change in 4th digit indicates a breaking change)
 # Change to 5.1.0.xx-SNAPSHOT to pick up maven local published copy
-cordaApiVersion=5.1.0.31-beta+
+cordaApiVersion=5.1.0.32-alpha-1696952014426
```
Comment (filipesoliveira) [inline gradle.properties]: The API version must be updated before merging the PR!

**Comment by filipesoliveira** [comment]
Probably it is best to remove the deprecated code instead of suppressing the warnings.

File: testing/cpbs/packaging-verification-app-v1/src/main/kotlin/com/r3/corda/testing/packagingverification/TransferStatesFlow.kt [hunk 2]
```diff
@@ -115,11 +115,13 @@ class TransferStatesFlow : ClientStartableFlow {
             log.info("Finalizing transaction")
             utxoLedgerService.finalize(signedTransaction, listOf(session))
             // Release the claim on the tokens' states, indicating we spent them all
+            @Suppress("DEPRECATION")
```
Comment (vkolomeyko) [inline testing/cpbs/packaging-verification-app-v1/src/main/kotlin/com/r3/corda/testing/packagingverification/TransferStatesFlow.kt]: I think we should remove use of this method.

**Review by vkolomeyko — DISMISSED** [review]
LGTM

**Comment by vkolomeyko** [comment]
Detekt complains:

<html>
<body>


/home/jenkins/agent/workspace/Corda5_corda-runtime-os_PR-4826/testing/cpbs/packaging-verification-app-v1/src/main/kotlin/com/r3/corda/testing/packagingverification/TransferStatesFlow.kt:16:1: The import 'net.corda.v5.ledger.utxo.token.selection.TokenClaim' is unused. [UnusedImports] |  
-- | --
  |   |  
  | /home/jenkins/agent/workspace/Corda5_corda-runtime-os_PR-4826/testing/cpbs/packaging-verification-app-v1/src/main/kotlin/com/r3/corda/testing/packagingverification/TransferStatesFlow.kt - 5min debt |  
  | [33mUnusedImports - [The import 'net.corda.v5.ledger.utxo.token.selection.TokenClaim' is unused.] at /home/jenkins/agent/workspace/Corda5_corda-runtime-os_PR-4826/testing/cpbs/packaging-verification-app-v1/src/main/kotlin/com/r3/corda/testing/packagingverification/TransferStatesFlow.kt:16:1[0m


</body>
</html>

**Review by owenstanford — DISMISSED** [review]

**Review by owenstanford — APPROVED** [review]

**Review by vkolomeyko — APPROVED** [review]
LGTM

**Review by lankydan — APPROVED** [review]

**Pull request merged**